.github/workflows/e2e.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         with:
           api-level: 29
           # All the tests need to be run in this script as the emulator is killed after this command.
-          script: cd sample && react-native run-android --variant=release && curl --output /dev/null --silent --head --fail http://127.0.0.1:4723/wd/hub/sessions && yarn test
+          script: cd sample && react-native run-android && curl --output /dev/null --silent --head --fail http://127.0.0.1:4723/wd/hub/sessions && yarn test
```

Comment:
https://github.com/react-native-community/upgrade-support/issues/38#issuecomment-1059532701

can't merge if this does not get fixed